actions/hybrid_branch_deploy/action.yml

```diff
@@ -60,7 +60,7 @@ runs:
         ssh: ${{ env.DOCKER_BUILD_SSH }}
         context: ${{ fromJson(inputs.location).directory }}
         push: true
-        tags: "${{ fromJson(inputs.location).registry }}:${{ github.sha }}"
+        tags: "${{ fromJson(inputs.location).registry }}:${{ github.sha }}-${{ github.run_id }}"
```

Comment:
https://docs.github.com/en/actions/learn-github-actions/variables
Just wanted to note:
"This number does not change if you re-run the workflow run. For example, 1658821493."

Still I think it should be ok unless they re-run from the runs page?